### PR TITLE
add instrumentation for sql params

### PR DESCRIPTION
**Pull request by kdoh**

### CODE REVIEW

**Comment by kdoh** [comment]
@tlrobinson Let me know if this works for you.

File: frontend/src/metabase/query_builder/template_tags/TagEditorHelp.jsx [hunk 1]
```diff
@@ -109,7 +120,7 @@ const TagEditorHelp = ({ setQuery, sampleDatasetId }) => {
             <TagExample datasetQuery={EXAMPLES.multipleOptional} setQuery={setQueryWithSampleDatasetId} />
 
             <p>
-                <a href="http://www.metabase.com/docs/latest/users-guide/start" target="_blank">Read the full documentation</a>
+                <a href="http://www.metabase.com/docs/latest/users-guide/start" target="_blank" onClick={() => MetabaseAnalytics.trackEvent('QueryBuilder', 'Template Tag Documentation Click') }>Read the full documentation</a>
```
Comment (tlrobinson) [inline frontend/src/metabase/query_builder/template_tags/TagEditorHelp.jsx]: Any reason not to use `data-metabase-event` in cases like this?

Comment (kdoh) [inline frontend/src/metabase/query_builder/template_tags/TagEditorHelp.jsx]: Nah just forgot that it existed. Is that preferred in a case like this?

File: frontend/src/metabase/query_builder/template_tags/TagEditorHelp.jsx [hunk 2]
```diff
@@ -47,12 +47,20 @@ const EXAMPLES = {
...
 }
 
+
 const TagExample = ({ datasetQuery, setQuery }) =>
     <div>
         <h5>Example:</h5>
         <p>
             <Code>{datasetQuery.native.query}</Code>
-            { setQuery && <div className="Button Button--small" onClick={() => setQuery(datasetQuery, true)}>Try it</div> }
+            { setQuery && (
```
Comment (tlrobinson) [inline frontend/src/metabase/query_builder/template_tags/TagEditorHelp.jsx]: Also removed `setQuery` here.

File: frontend/src/metabase/query_builder/template_tags/TagEditorHelp.jsx [hunk 3]
```diff
@@ -47,12 +47,20 @@ const EXAMPLES = {
...
+                <div
+                    className="Button Button--small"
+                    data-metabase-event="QueryBuilder;Template Tage Example Query Used"
```
Comment (tlrobinson) [inline frontend/src/metabase/query_builder/template_tags/TagEditorHelp.jsx]: s/Tage/Tag/

**Comment by tlrobinson** [comment]
LGTM

**Pull request merged**